app/3.0/docs/tools/polymer-cli-commands.md

```diff
@@ -232,11 +267,47 @@ App project demo:
 
 ### Server options {#server-options}
 
-This section shows examples of using various `polymer serve` options.
+This section describes command line options available for the Polymer CLI development server (`polymer serve`).
+
+### --npm
+
+Sets npm mode. Dependencies are installed from npm, the component directory is set to `node_modules` and the package name is read from `package.json`.
+
+```
+polymer serve --npm --module-resolution="node"
+```
+
+### --module-resolution
 
-Serve from port 3000:
+Specifies how to resolve module specifiers in import and export statements when rewriting them to be web-compatible.
 
-    polymer serve --port 3000
+Valid values are `"none"` and `"node"`. Defaults to `"none"`. 
+
+* `"none"` disables module specifier rewriting. 
+
+* `"node"` uses Node.js resolution to find modules. 
```

Comment:
We should mention that because Polymer projects: elements, LitElement, etc., use bare specifiers, that `--module-resolution=node` is generally required, and that it's much easier to just put these in polymer.json.